app/src/androidTest/java/com/github/giommok/softwaredevproject/DatabaseTest.java

```diff
@@ -3,69 +3,61 @@
-
-import com.google.firebase.FirebaseApp;
 
 import org.junit.After;
 import org.junit.Assert;
-import org.junit.Before;
-import org.junit.BeforeClass;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
+import static com.github.giommok.softwaredevproject.AccountTest.basicUsername;
+import static com.github.giommok.softwaredevproject.AccountTest.sleepTime;
 import static org.junit.Assert.*;
 
 
 import java.util.Arrays;
+import java.util.Collections;
+import java.util.Random;
 
 @RunWith(AndroidJUnit4.class)
 public class DatabaseTest {
     private static Context context;
 
-    /* Set up the environment */
-    @BeforeClass
-    public static void init() throws InterruptedException {
-        context = InstrumentationRegistry.getInstrumentation().getTargetContext();
-        FirebaseApp.initializeApp(context);
-
-        /* Make sure that mock users are not on the database before the tests*/
-        Database.refRoot.child("users").child("null").removeValue();
-        Thread.sleep(1500);
-    }
+    private static final int userOffset = new Random().nextInt();
+    private static final String user1 = basicUsername + userOffset;
 
     /* Make sure that mock users are not on the database after a test */
     @After
     public void removeTestUsers() throws InterruptedException {
-        Database.refRoot.child("users").child("null").removeValue();
-        Thread.sleep(1500);
+        Database.refRoot.child(Database.CHILD_USERS).child(user1).removeValue();
+        Thread.sleep(sleepTime);
     }
 
     /* Test that isPresent method works */
     @Test
     public void isPresentTest() throws InterruptedException {
-        Database.setChild("users/null", Arrays.asList("username"), Arrays.asList("username@Test"));
-        Thread.sleep(1500);
-        Database.isPresent("users", "email", "dota2>lol", Assert::fail, () -> assertTrue("Correct behavior", true));
-        Database.isPresent("users", "username", "username@Test", () -> assertTrue("Correct behavior", true),  Assert::fail);
-        Thread.sleep(1500);
+        Database.setChild(Database.CHILD_USERS + user1, Collections.singletonList(Database.CHILD_USERNAME), Collections.singletonList(user1));
+        Thread.sleep(sleepTime);
+
+        Database.isPresent(Database.CHILD_USERS, Database.CHILD_USERNAME, "", Assert::fail, () -> assertTrue("Correct behavior", true));
+        Database.isPresent(Database.CHILD_USERS, Database.CHILD_USERNAME, user1, () -> assertTrue("Correct behavior", true),  Assert::fail);
+        Thread.sleep(sleepTime);
     }
 
     /* Test that setChild method works */
     @Test
     public void setChildTest() throws InterruptedException {
-        Database.isPresent("users", "username", "testing@Value", Assert::fail, () -> assertTrue("Correct behavior", true));
+        Database.isPresent(Database.CHILD_USERS, Database.CHILD_USERNAME, user1, Assert::fail, () -> assertTrue("Correct behavior", true));
         Thread.sleep(1500);

```

Comment:
Replace value by the newly defined constant
```suggestion
        Thread.sleep(SLEEP_TIME);
```